### ISSUE TITLE
iTunes Downloads listview enhancement

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2641]
**Reported by jpgreer on 2012-09-04 11:20**
This is kind of a minor thing for the iTunes app module, but as it is NVDA does not read currently downloading items in iTunes without first moving away from the listview, then NVDA reads it. It does not read as you arrow up and down the list however. I've included object info below that may help this issue.
Developer info for navigator object:
name: u'movie: Star Trek I: The Motion Picture / Robert Wise Status: 667.8 MB of 1.77 GB - 32 minutes remaining '
role: ROLE_LISTITEM
states: STATE_SELECTABLE, STATE_FOCUSED, STATE_SELECTED
Python object: <NVDAObjects.IAccessible.IAccessible object at 0x04FD95F0>
Python class mro: (<class 'NVDAObjects.IAccessible.IAccessible'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <type 'object'>)
description: None
location: (187, 160, 1717, 34)
value: None
appModule: <'itunes' (appName u'itunes', process ID 292) at address 51bc370>
TextInfo: <class 'NVDAObjects.NVDAObjectTextInfo'>
windowHandle: 2165786
windowClassName: u'iTunesDownloadList'
windowControlID: 244653552
windowStyle: 1375797248
windowThreadID: 1432
windowText: u'downloads'
IAccessibleObject: <POINTER(IAccessible) ptr=0x86796f8 at 4eddd00>
IAccessibleChildID: 3
IAccessible event parameters: windowHandle=2165786, objectID=None, childID=3
IAccessible accRole: ROLE_SYSTEM_LISTITEM
IAccessible accState: STATE_SYSTEM_SELECTABLE, STATE_SYSTEM_SELECTED, STATE_SYSTEM_FOCUSED, STATE_SYSTEM_VALID (2097158)

**Comment by leonardder** [comment]
@jpgreer, are you the jpgreer that opened this issue? If so, could you tell whether this is still happening for you?

**Comment by leonardder** [comment]
Closing, as the original author hasn't responded to this in a long while.
